pkg/workflows/wasm/sdk/runner_wasip1.go

```diff
@@ -18,11 +21,25 @@ var (
 //go:wasmimport env sendResponse
 func sendResponse(respptr unsafe.Pointer, respptrlen int32)
 
+//go:wasmimport env log
+func log(respptr unsafe.Pointer, respptrlen int32)
+
 func bufferToPointerLen(buf []byte) (unsafe.Pointer, int32) {
 	return unsafe.Pointer(&buf[0]), int32(len(buf))
 }
 
 func NewRunner() *Runner {
+
+	wws := &WasmWriteSyncer{
+		logFn: func(msg string) {
+			ptr, ptrlen := bufferToPointerLen(bytes.NewBufferString(msg).Bytes())
+			log(ptr, ptrlen)
+			return
+		},
+	}
+	core := zapcore.NewCore(zapcore.NewJSONEncoder(zap.NewProductionEncoderConfig()), zapcore.AddSync(wws), zapcore.InfoLevel)
```

Comment:
@agparadiso Note, calling AddSync means you don't need to implement Sync on line 76, so I would just implement the Write method down below